sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -1165,6 +1178,17 @@ public GenericRecord apply(GenericRecord input) {
     private GenericRecordPassthroughFn() {}
   }
 
+  /** Encapsulates checked exceptions with Parquet file operation as RuntimeException. */
+  public static class ParquetIoException extends RuntimeException {
```

Comment:
I know this is going to sound a bit odd but can we just remove this and let exception handling as it was before. Beam rarely does this kind of wrapping and I don't want ParquetIO to differ from how others IOs behave.